### PR TITLE
Updated codebase with some fixes.

### PR DESCRIPTION
**Pull request by amar-c8y**
Remove some unwanted import and added few fixes.

### CODE REVIEW

File: asset-properties-widget/component/asset-properties-config/asset-properties.config.component.spec.ts
Comment (ghost) [inline asset-properties-widget/component/asset-properties-config/asset-properties.config.component.spec.ts:1]: rename the component file as asset-properties-config.component.spec.ts

**Review by ghost — CHANGES_REQUESTED** [review]

File: asset-properties-widget/component/asset-properties-config/asset-properties.config.component.spec.ts
Comment (amar-c8y) [reply asset-properties-widget/component/asset-properties-config/asset-properties.config.component.spec.ts:1]: modified

File: package.json [hunk 1]
```diff
@@ -69,10 +69,10 @@
   "c8y": {
     "application": {
       "name": "Asset properties widget",
-      "description": "Asset properties Configuration widget",
-      "contextPath": "asset-properties",
-      "key": "asset-properties-application-key",
-      "globalTitle": "Asset properties Configuration widget",
+      "description": "Asset properties configuration widget",
```
Comment (ghost) [inline package.json]:
```suggestion
      "description": "Asset properties widget",
```

File: package.json [hunk 2]
```diff
@@ -69,10 +69,10 @@
...
+      "contextPath": "asset-properties-widget",
+      "key": "asset-properties-widget-application-key",
+      "globalTitle": "Asset properties configuration widget",
```
Comment (ghost) [inline package.json]:
```suggestion
      "globalTitle": "Asset properties widget",
```

**Review by ghost — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Review by spau6 — APPROVED** [review]

**Pull request merged**